crates/router/src/core/fraud_check/operation/fraud_check_post.rs

```diff
@@ -477,25 +477,34 @@ impl<F: Clone + Send> UpdateTracker<FrmData, F> for FraudCheckPost {
         };
 
         if let Some(frm_suggestion) = frm_suggestion {
-            let (payment_attempt_status, payment_intent_status) = match frm_suggestion {
-                FrmSuggestion::FrmCancelTransaction => {
-                    (AttemptStatus::Failure, IntentStatus::Failed)
-                }
-                FrmSuggestion::FrmManualReview => (
-                    AttemptStatus::Unresolved,
-                    IntentStatus::RequiresMerchantAction,
-                ),
-                FrmSuggestion::FrmAuthorizeTransaction => {
-                    (AttemptStatus::Authorized, IntentStatus::RequiresCapture)
-                }
-            };
+            let (payment_attempt_status, payment_intent_status, merchant_decision, error_message) =
+                match frm_suggestion {
+                    FrmSuggestion::FrmCancelTransaction => (
+                        AttemptStatus::Failure,
+                        IntentStatus::Failed,
+                        Some(MerchantDecision::Rejected.to_string()),
+                        Some(Some(CANCEL_INITIATED.to_string())),
+                    ),
+                    FrmSuggestion::FrmManualReview => (
+                        AttemptStatus::Unresolved,
+                        IntentStatus::RequiresMerchantAction,
+                        None,
+                        None,
+                    ),
+                    FrmSuggestion::FrmAuthorizeTransaction => (
```

Comment:
Can we rename this to `FrmCaptureTransaction`?

Reply:
When manual capture payment which was marked as fraud and held, when approved needs to be authorized. FrmCaptureTransaction seems to be capture the payment whereas we are actually authorizing the payment.